apps/dataset-browser/src/messages/nl/messages.json

```diff
@@ -68,9 +69,52 @@
       "dateCreated": "Datum gecreëerd",
       "dateModified": "Datum gewijzigd",
       "keywords": "Trefwoorden"
+    },
+    "measurements": {
+      "title": "Transparantiemetingen",
+      "moreInfo": "Meer informatie over de transparantiemetingen en hoe u deze kunt verbeteren."
     }
   },
   "Register": {
     "title": "Registreer dataset"
+  },
+  "TransparencyMetrics": {
+    "unknown": "Onbekend",
+    "https%3A%2F%2Fdata%2Ecolonialcollections%2Enl%2Fmetrics%23open-license": {
+      "shortTitle": "Open licentie",
+      "longTitle": "Open licentie",
+      "descriptionTrue": "De dataset is beschikbaar met een open licentie.",
+      "descriptionFalse": "De dataset is niet beschikbaar met een open licentie."
+    },
+    "https%3A%2F%2Fdata%2Ecolonialcollections%2Enl%2Fmetrics%23languages": {
+      "shortTitle": "Talen",
+      "longTitle": "Talen",
+      "descriptionTrue": "De dataset maakt duidelijk in welke taal/talen deze beschikbaar is.",
+      "descriptionFalse": "De dataset maakt niet duidelijk in welke taal/talen deze beschikbaar is."
+    },
+    "https%3A%2F%2Fdata%2Ecolonialcollections%2Enl%2Fmetrics%23online": {
+      "shortTitle": "Online",
+      "longTitle": "Downloadbaar of online toegankelijk",
+      "descriptionTrue": "De dataset kan worden gedownload of is online toegankelijk.",
+      "descriptionFalse": "De dataset kan niet worden gedownload en is niet online toegankelijk."
+    },
+    "https%3A%2F%2Fdata%2Ecolonialcollections%2Enl%2Fmetrics%23structured-format": {
+      "shortTitle": "Gestructureerd formaat",
+      "longTitle": "Gestructureerd formaat",
+      "descriptionTrue": "De dataset is beschikbaar in een gestructureerd formaat (bijv. Excel).",
+      "descriptionFalse": "De dataset is niet beschikbaar in een gestructureerd formaat (bijv. Excel)."
+    },
+    "https%3A%2F%2Fdata%2Ecolonialcollections%2Enl%2Fmetrics%23open-format": {
+      "shortTitle": "Open formaat",
+      "longTitle": "Open formaat",
+      "descriptionTrue": "De dataset is beschikbaar in een open formaat (bijv. door komma's gescheiden waarden).",
+      "descriptionFalse": "De dataset is niet beschikbaar in een open formaat (bijv. door komma's gescheiden waarden)."
+    },
+    "https%3A%2F%2Fdata%2Ecolonialcollections%2Enl%2Fmetrics%23rdf-format": {
+      "shortTitle": "RDF formaat",
+      "longTitle": "RDF formaat",
+      "descriptionTrue": "De dataset is beschikbaar in RDF-formaat.",
```

Comment:
Very minor: `in een RDF-formaat` (to be consistent with the other metric descriptions).